l1-contracts/src/mock/StakingAssetHandler.sol

```diff
@@ -157,6 +166,30 @@ contract StakingAssetHandler is IStakingAssetHandler, Ownable {
     }
   }
 
+  /**
+   * Re add a validator that has already supplied a passport proof.
+   * Used to re-enter a validator that has been exited during testnet.
+   *
+   * @param _attester - the validator's attester address
+   */
+  function reenterExitedValidator(address _attester, address _proposer)
```

Comment:
This can be griefed. 

Only possible to add if there `< depositsPerMint` in the queue, as the only way to get funding is calling the `dripQueue` and `reenterExitedValidator` can only use leftovers.

If there are a lot of people in queue, you will never be readded because it is trying to use the same funds.

It should either:
- add them to the queue
- mint funds and add them directly

I would say queue, because it seems kinda weird otherwise. e.g., say you were kicked because you are not active or malicuious, so slashing kicked in. But now you are just in there again back to the same business. And you could at the same time bypass the queue 👀 (I guess with the "grief" you can practically only get in when there are no queue)